### PR TITLE
Attribution

### PR DESCRIPTION
**Pull request by jklmnn**
Added attribution.

### CODE REVIEW

File: park_api/env.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ def determine_server_version():
     try:
         proc = subprocess.Popen(["git", "rev-list", "--all", "--count"], stdout=subprocess.PIPE)
         rev = proc.stdout.read().decode('utf-8').strip()
-        SERVER_VERSION = '0.1.{0}'.format(rev)
+        SERVER_VERSION = '0.2.{0}'.format(rev)
```
Comment (kiliankoe) [inline park_api/env.py]: Didn't we decide on incrementing this with breaking changes? The iOS version will not accept server responses with version != 0.1.x... I should probably change that.

Comment (jklmnn) [inline park_api/env.py]: That was for the API version, not the server version.

Comment (kiliankoe) [inline park_api/env.py]: Oh, I see. Nevermind my comment then :)

**Comment by jklmnn** [comment]
If no further comment will come, I'll merge tomorrow.

**Pull request merged**